qdrant-landing/content/documentation/cloud/create-cluster.md

```diff
@@ -5,62 +5,96 @@ weight: 20
+
+You can always upgrade to a standard cluster with more resources and features.
+
+## Standard Clusters
+
+On top of the Free cluster features, Standard clusters offer:
+
+- Response time and uptime SLAs
+- Dedicated resources
+- Backup and disaster recovery
+- Multi-node clusters for high availability
+- Horizontal and vertical scaling
+- Monitoring and log management
+- Zero-downtime upgrades for multi-node clusters with replication
+
+You have a broad choice of regions on AWS, Azure and Google Cloud.
+
+For payment information see [**Pricing and Payments**](/documentation/cloud/pricing-payments/).
+
+## Create a cluster
+
 This page shows you how to use the Qdrant Cloud Console to create a custom Qdrant Cloud cluster.
 
 > **Prerequisite:** Please make sure you have provided billing information before creating a custom cluster. 
 
 1. Start in the **Clusters** section of the [Cloud Dashboard](https://cloud.qdrant.io/). 
 1. Select **Clusters** and then click **+ Create**.
 1. In the **Create a cluster** screen select **Free** or **Standard**
-   For more information on a free cluster, follow [Free Tier Setup](/documentation/cloud/qdrant-cloud-setup/). The remaining steps assume you want a standard cluster.
+   Most of the remaining configuration options are only available for standard clusters.
 1. Select a provider. Currently, you can deploy to:
 
    - Amazon Web Services (AWS)
    - Google Cloud Platform (GCP)
    - Microsoft Azure 
+   - Your own [Hybrid Cloud](/documentation/hybrid-cloud/) Infrastructure
 
-1. Choose your data center region. If you have latency concerns or other topology-related requirements, [**let us know**](mailto:cloud@qdrant.io).
-1. Configure RAM for each node (2 GB to 64 GB). 
-   >  For more information, see our [**Capacity and Sizing**](/documentation/cloud/capacity-sizing/) guidance. If you need more capacity per node, [**let us know**](mailto:cloud@qdrant.io).
-1. Choose the number of vCPUs per node (0.5 core to 16 cores). If you add more
+1. Choose your data center region or Hybrid Cloud environment. 
+1. Configure RAM for each node. 
+   >  For more information, see our [**Capacity and Sizing**](/documentation/cloud/capacity-sizing/) guidance.
+1. Choose the number of vCPUs per node. If you add more
    RAM, the menu provides different options for vCPUs.
-1. Select the number of nodes you want the cluster to be deployed on. 
-
-   > Each node is automatically attached with a disk space offering enough space for your data if you decide to put the metadata or even the index on the disk storage.
-1. Select the disk space for your deployment. You can choose from 8 GB to 2 TB.
+1. Select the number of nodes you want the cluster to be deployed on.
+   > Each node is automatically attached with a disk space offering enough space for your data for a default setup.
+1. Select additional disk space for your deployment.
+   > This can be required, if you want to store all data on disk and only use RAM for caching.
```

Comment:
It depends on the collection configuration. If use RAM only for caching, the disk to RAM ration may need to be different to the default one.

Reply:
I also just realised theres another 'a disk space' on line 80

Reply:
But either way it reads weirdly as you're using an indefinite verb to amend a definite verb, and following it up with a conditional 

It either is required or isn't based on the conditional as a qualifier.

'It would be recommended, if' suits a more ambiguous suggestion

Reply:
If you kept the 'can be required' line, I would be anticipating another clause which explained the trade-off making the use of 'can' which offers a choice make sense

'It can be required, if ... but this comes at a higher resource cost.'

Reply:
Rephrased it to

> Depending on your collection configuration, you may need more disk space per RAM. For example, if you configure `on_disk: true` and only use RAM for caching.

to make this clearer.